transformer_engine/common/fused_attn/fused_attn.cpp

```diff
@@ -329,11 +345,17 @@ void nvte_fused_attn_bwd_qkvpacked(
   } else if (fused_attention_backend == NVTE_Fused_Attn_Backend::NVTE_F16_arbitrary_seqlen) {
 #if (CUDNN_VERSION >= 8900)
       Tensor *output_S = reinterpret_cast<Tensor *>(Aux_CTX_Tensors->tensors[0]);
-      const Tensor *input_rng_state = reinterpret_cast<const Tensor*>(Aux_CTX_Tensors->tensors[1]);
+      Tensor *input_Bias, *input_rng_state;
+      if ((bias_type != NVTE_NO_BIAS) && (bias_type != NVTE_ALIBI)) {
+          input_Bias = reinterpret_cast<Tensor*>(Aux_CTX_Tensors->tensors[1]);
+          input_rng_state = reinterpret_cast<Tensor*>(Aux_CTX_Tensors->tensors[2]);
```

Comment:
Could we keep `input_rng_state` in the Aux_CTX_Tensors->tensors[1] to align the other cases?

Reply:
Done. I tried to keep `rng_state` last because our FP8 API is also having it last. Also, the bias tensor is part of the Aux tensor pack because it's flash and bias is required to recompute softmax, so my intention was to keep all the softmax-related tensors together. But if it makes it difficult for Jax to implement, I've swapped them. Thanks.